specification/schemas/types/LinksError.yaml

```diff
@@ -3,8 +3,7 @@ additionalProperties: false
 properties:
   about:
     type: string
-    enum:
-      - 'https://digital.nhs.uk/developer/api-catalogue/communications-manager'
     format: uri
     example: 'https://digital.nhs.uk/developer/api-catalogue/communications-manager'
```

Comment:
Although there's currently a redirect in place, can you update these to the new nhs-notify url please? e.g. `https://digital.nhs.uk/developer/api-catalogue/nhs-notify`

Reply:
Actually, given that this is an example of a response message that we return, can you check whether we've updated the urls in our actual responses yet - if we haven't then it's fine to leave these pointing to the old urls

Reply:
Investigated by David - The API response still returns the old `communications-manager` path so the docs are correct that they match this. The urls will be updated by CCM-3068